modular_skyrat/modules/modular_vending/code/vending.dm

```diff
@@ -14,8 +14,38 @@
 	if(skyrat_contraband)
 		contraband += skyrat_contraband
 
+	/// Time to make clothes amounts consistent!
+	for (var/item in products)
+		if(products[item] < 5 && ispath(item, /obj/item/clothing) && skyrat_IsNotGameplay(item))
```

Comment:
That `5` should be put into a define instead, something like `#define MINIMUM_CLOTHING_STOCK 5` at the start of the file, and then replacing all of those `5` with `MINIMUM_CLOTHING_STOCK` and finally adding at the end of the file a `#undef MINIMUM_CLOTHING_STOCK`.

Reply:
I did really debate this, and this was part of the original code, but I don't see too much of a reason for things to have this number overridden, but I'll add it for customization's sake!